.editorconfig

```diff
@@ -0,0 +1,16 @@
+# editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = tab
```

Comment:
I find using tabs nice because in a lot of editors you can set the size of tabs (ie hard tabs that are 2 wide, which is how I have it) but if spaces are preferred I can make the change throughout the files.